contracts/proxy/transparent/ProxyAdmin.sol

```diff
@@ -9,14 +9,16 @@ import {Ownable} from "../../access/Ownable.sol";
 /**
  * @dev This is an auxiliary contract meant to be assigned as the admin of a {TransparentUpgradeableProxy}. For an
  * explanation of why you would want to use this see the documentation for {TransparentUpgradeableProxy}.
+ *
+ * NOTE: This contract does not make use of {Context} deliberately.
```

Comment:
I think this comment can be removed from this contract. There is no use of `msg.sender` (except through Ownable, which does use Context).

Reply:
I added it by accident. Removed